### DISCUSSION TITLE
Rust stats

### DISCUSSION THREAD

**Discussion by Qodestackr** [anuraghazra/github-readme-stats#2743] [Q&A] [answered]
How will I make it detect my rust repos? I have a private project but it seems to never appear on the stats, is it only public repos?

**Comment by Qodestackr** [comment]
I have:
<img width="400px" align="left" src="https://github-readme-stats.vercel.app/api/top-langs/?username=Qodestackr&hide=css,html,SCSS&count_private=true&theme=synthwave&layout=compact"/></td>

**Reply by Qodestackr** [reply]
I mean `<img width="400px" align="left" src="https://github-readme-stats.vercel.app/api/top-langs/?username=Qodestackr&hide=css,html,SCSS&count_private=true&theme=synthwave&layout=compact"/></td>`

**Comment by qwerty541** [comment, marked as answer]
Top languages card show usage stats only for public non-forked repositories. I have several public rust repositories and top languages card show it as well. You can check it in my profile.

**Reply by rickstaa** [reply]
@qwerty541 is correct if you want results of your private repositories to show up, you have to [deploy your own Vercel instance](https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own).

**Comment by Qodestackr** [comment]
Thanks, enough to close this :+1:
